types/chai/index.d.ts

```diff
@@ -791,7 +791,7 @@ declare namespace Chai {
          * @param value   Actual value.
          * @param message   Message to display on error.
          */
-        isBoolean<T>(value: T, message?: string): void;
+        isBoolean<T>(value: T, message?: string): asserts value is (T extends boolean ? T : never);
```

Comment:
Is this going to assert `value is never` when `T` is `unknown`?

Reply:
You're right. That's not correct. 🤔 Let me try to improve that.